Problems installing mote

I was thiking about working on some of mote issues. I forked/cloned the repo, created a virtualenv and ran `python setup.py develop` but I got this error after some packages(including `python-openid-teams`) were successfully installed:

```
Searching for python-openid                          
Downloading https://pypi.python.org/packages/ce/a2/ae37cd9fd9067b0a01f22ca6261d0b4fbdb0b2398e20f4f00302c8cf5dff/python-openid-teams-1.1.tar.gz#md5=c8d30171a4f840a7dd9df0a1ef370fcc                                  
Best match: python-openid teams-1.1                  
Processing python-openid-teams-1.1.tar.gz            
Writing /tmp/easy_install-9BsUb2/python-openid-teams-1.1/setup.cfg                                        
Running python-openid-teams-1.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-9BsUb2/python-openid-teams-1.1/egg-dist-tmp-915pfI                                                                                
Removing /home/serafim/.virtualenvs/dev_mote/lib/python2.7/site-packages/python_openid_teams-1.1-py2.7.egg                                                                                                           
Moving python_openid_teams-1.1-py2.7.egg to /home/serafim/.virtualenvs/dev_mote/lib/python2.7/site-packages                                                                                                          
python-openid-teams 1.1 is already the active version in easy-install.pth                                 

Installed /home/serafim/.virtualenvs/dev_mote/lib/python2.7/site-packages/python_openid_teams-1.1-py2.7.egg                                                                                                          
error: The 'python-openid' distribution was not found and is required by mote
```
For some reason python-openid-teams is processed twice instead of python-openid.
This is the list of packages that were properly installed:

```
$ pip freeze
fedmsg==1.0.1
-e git+git@github.com:tserafim/mote.git@c0720557ee8a6a5ad6c44296f50a2448ae25e50f#egg=mote
python-openid-cla==1.2
python-openid-teams==1.1
requests==2.18.4
```

If I run `python setup.py develop` again, `python-opendid-teams` is properlly installed but i get another error during `cryptography` package installation.
```
Searching for cryptography
Reading https://pypi.python.org/simple/cryptography/
Downloading https://pypi.python.org/packages/bf/da/6a9f49cc7a970380c8235b3adab0c08c7c3d4814876f7383b33e3882a577/cryptography-2.1.1.tar.gz#md5=fa98118b468020349a798776aac6d572
Best match: cryptography 2.1.1
Processing cryptography-2.1.1.tar.gz
Writing /tmp/easy_install-xCvzkm/cryptography-2.1.1/setup.cfg
Running cryptography-2.1.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-xCvzkm/cryptography-2.1.1/egg-dist-tmp-MMQN9p
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'virtual:world', not found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'virtual:world', not found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'virtual:world', not found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'virtual:world', not found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'virtual:world', not found
c/_cffi_backend.c:15:10: fatal error: ffi.h: No such file or directory
 #include <ffi.h>
          ^~~~~~~
compilation terminated.
error: Setup script exited with error: command 'gcc' failed with exit status 1
```

After some search I found that I need to have `libffi-devel` installed on Fedora. Maybe mention this on the docs?

After isntalling `libffi-devel` I ran `python setup.py develop` again but got another error during installation of `pycparser` package:
```
Searching for pycparser
Reading https://pypi.python.org/simple/pycparser/
Downloading https://pypi.python.org/packages/8c/2d/aad7f16146f4197a11f8e91fb81df177adcc2073d36a17b1491fd09df6ed/pycparser-2.18.tar.gz#md5=72370da54358202a60130e223d488136
Best match: pycparser 2.18
Processing pycparser-2.18.tar.gz
Writing /tmp/easy_install-geDfgH/cryptography-2.1.1/temp/easy_install-9En2pG/pycparser-2.18/setup.cfg
Running pycparser-2.18/setup.py -q bdist_egg --dist-dir /tmp/easy_install-geDfgH/cryptography-2.1.1/temp/easy_install-9En2pG/pycparser-2.18/egg-dist-tmp-wKkogP
warning: no previously-included files matching 'yacctab.*' found under directory 'tests'
warning: no previously-included files matching 'lextab.*' found under directory 'tests'
warning: no previously-included files matching 'yacctab.*' found under directory 'examples'
warning: no previously-included files matching 'lextab.*' found under directory 'examples'
zip_safe flag not set; analyzing archive contents...
pycparser.ply.lex: module references __file__
pycparser.ply.lex: module MAY be using inspect.getsourcefile
pycparser.ply.yacc: module references __file__
pycparser.ply.yacc: module MAY be using inspect.getsourcefile
pycparser.ply.yacc: module MAY be using inspect.stack
pycparser.ply.ygen: module references __file__
creating /tmp/easy_install-geDfgH/cryptography-2.1.1/.eggs/pycparser-2.18-py2.7.egg
Extracting pycparser-2.18-py2.7.egg to /tmp/easy_install-geDfgH/cryptography-2.1.1/.eggs

Installed /tmp/easy_install-geDfgH/cryptography-2.1.1/.eggs/pycparser-2.18-py2.7.egg
no previously-included directories found matching 'docs/_build'
warning: no previously-included files matching '*' found under directory 'vectors'
build/temp.linux-x86_64-2.7/_openssl.c:483:10: fatal error: openssl/opensslv.h: No such file or directory
 #include <openssl/opensslv.h>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
error: Setup script exited with error: command 'gcc' failed with exit status 1
```

--- a/README.md
+++ b/README.md
@@ -18,7 +18,9 @@ Dependencies:
  - Optional: `memcached` (`sudo dnf install memcached`)
  - Python 2.7.x
 
-`pip` Dependencies:
+Installation:
+ - Optional: create virtual environment
+ - pip install -r requirements.txt
  - Install using `python setup.py develop`
 
 Running møte:

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ Dependencies:
 
 Installation:
  - Optional: create virtual environment
- - pip install -r requirements.txt
+ - `pip install -r requirements.txt`
  - Install using `python setup.py develop`
 
 Running møte:

